F/FFLAS_FFPACK/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "FFLAS_FFPACK"
+version = v"2.5.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/linbox-team/fflas-ffpack/releases/download/v$version/fflas-ffpack-$version.tar.gz", "dafb4c0835824d28e4f823748579be6e4c8889c9570c6ce9cce1e186c3ebbb23"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/fflas-ffpack-*
+
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+
+autoreconf
+./configure CCNAM=${CC} CPLUS_INCLUDE_PATH=$includedir --prefix=$prefix --build=${MACHTYPE} --host=${target}
+
+make -j ${nproc}
+make install
+
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(exclude=Sys.iswindows) |> expand_cxxstring_abis
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("fflas-ffpack-config", :fflas_ffpack_config)
```

Comment:
This is fairly useless, it's just the config script (which in any case shouldn't be an `ExecutableProduct` since it isn't a binary executable).
```suggestion
```